ember-cli-build.js

```diff
@@ -29,7 +29,12 @@ module.exports = function(defaults) {
         'ember-prism': {
             theme: 'twilight',
             components: highlightedLanguages,
-        }
+        },
+        sassOptions: {
+            includePaths: [
+                'node_modules/normalize.css',
```

Comment:
where is the `normalize.css` package coming from? don't we have to add a dependency?

Reply:
nevermind, I'm blind... just found it 🙈